### DISCUSSION TITLE
Self Approved requests from admin

### DISCUSSION THREAD

**Discussion by moamen-gamal** [Keyfactor/ejbca-ce#209] [Q&A] [answered]
I want to make a request( create certificate / revoke / edit ) self approved if the one who makes is a super admin without approval from another admin but i can't find a way to make that so i'm stuck with making multiple certificate profile.
Thanks in advance.

**Comment by mike-agrenius-kushner** [comment, marked as answer]
Hi,

I'm afraid that's not possible atm – we've had some feature requests based on role based override of approvals, but it's not currently in the near pipeline. 

Cheers,
Mike
